src/Reflection/ReflectionClass.php

```diff
@@ -79,12 +79,15 @@ public static function export($className = null)
 
     public function __toString()
     {
-        $format  = "Class [ <user> class %s%s%s ] {\n";
+        $isObject = $this instanceof ReflectionObject;
+
+        $format  = "%s [ <user> class %s%s%s ] {\n";
         $format .= "  @@ %s %d-%d\n\n";
         $format .= "  - Constants [%d] {%s\n  }\n\n";
         $format .= "  - Static properties [%d] {%s\n  }\n\n";
         $format .= "  - Static methods [%d] {%s\n  }\n\n";
         $format .= "  - Properties [%d] {%s\n  }\n\n";
+        $format .= ($isObject ? "  - Dynamic properties [%d] {%s\n  }\n\n" : '%s%s');
```

Comment:
Actually, logic should be replaced in the child class with some duplication. Horrible but correct solution :( 

I'm fine with having a `@TODO` instead of that